### ISSUE TITLE
Cannot start gateway with passphrase is a number

### ISSUE THREAD

**Issue by sytranvn** [hummingbot/hummingbot#6057]
**Describe the bug**
Start gateway with `--passphrase` option is a number cause error.

**Steps To Reproduce**
1. Generate certificate with passphrase is a number (eg: 4321)
2. Start gateway `yarn dev --passphrase=4321`
3. Gateway fail to start with error
```
2023-02-03 08:52:52 | info |    ⚡️ Starting Gateway API on port 15888...
2023-02-03 08:52:52 | error |   Failed to start the server with https. Confirm that the SSL certificate files exist and are correct. Error: TypeError [ERR_INVALID_ARG_TYPE]: The "options.passphrase" property must be of type string. Received type number (4321)
```

**Screenshots**
// If applicable, add screenshots to help explain your problem.
![Screen Shot 2023-02-03 at 14 47 05](https://user-images.githubusercontent.com/110009581/216541953-b18cf5fd-0e81-4b45-b11b-ce09750e743f.png)

**Release version**
// Include your bot's version number (Can be found at the upper left corner of your CLI).
1.12.0
**Attachments**
// Please attach your config file and log file to help us debug. It would be difficult for us to help you without those! 

*WARNING: Do NOT publish any exchange API keys or your wallet's private key. Whoever has access to them may steal your assets!*

**Optional: your discord username:**

**Comment by sytranvn** [comment]
Can I make a PR to fix this?

**Comment by rapcmia** [comment]
Hi @sytranvn good day to you, 
Please be advised our team is working on some changes on gateway but we dont have yet the timeframe of when it would be available for public. At the moment we would recommend to use strings as your passphrase but also you can submit a PR for a fix for the current gateway version. Thank you for time

**Comment by sytranvn** [comment]
Thanks @rapcmia , I have submitted #6067 to fix this issue.

**Comment by rapcmia** [comment]
Hi @sytranvn thanks for the bugfix ticket. It is now merged development and available on the next version `1.13.0` 🙇
